packages/boxel-ui/test-app/tests/integration/components/resizable-panel-group-horizontal-test.gts

```diff
@@ -0,0 +1,310 @@
+
+            <ResizablePanel
+              @defaultLengthFraction={{panel.defaultLengthFraction}}
+              @lengthPx={{panel.lengthPx}}
+              @minLengthPx={{panel.minLengthPx}}
+              @isHidden={{panel.isHidden}}
+            >
+              {{#if (eq index 1)}}
+                <div
+                  class='panel-{{index}}-content'
+                  style={{panel.outerContainerStyle}}
+                >
+                  <div>
+                    Panel
+                    {{index}}
+                  </div>
+                </div>
+              {{else}}
+                <div
+                  class='panel-{{index}}-content'
+                  style={{panel.outerContainerStyle}}
+                >
+                  Panel 2
+                </div>
+              {{/if}}
+            </ResizablePanel>
+            {{#if panel.showResizeHandle}}
+              <ResizeHandle />
+            {{/if}}
+          {{/each}}
+        </ResizablePanelGroup>
+      </div>
+    </template>);
+    await sleep(100); // let didResizeModifier run
+  }
+
+  test<MyTestContext>('it can lay out panels horizontally', async function (assert) {
+    this.renderController.containerStyle =
+      'max-width: 100%; height: 100px; width: 318px;';
+    await renderHorizontalResizablePanelGroup(this.renderController);
+    assert.hasNumericStyle('.panel-0-content', 'width', 300 * 0.6, 1);
+    assert.hasNumericStyle('.panel-1-content', 'width', 300 * 0.4, 1);
+  });
+
+  test<MyTestContext>('it can lay out panels horizontally (length specified)', async function (assert) {
+    this.renderController.containerStyle =
+      'max-width: 100%; height: 200px; width: 518px; border: 1px solid green';
+    this.renderController.panels[0].lengthPx = 355;
+    this.renderController.panels[1].lengthPx = 143;
+    await renderHorizontalResizablePanelGroup(this.renderController);
+    assert.hasNumericStyle('.panel-0-content', 'width', 355, 1);
+    assert.hasNumericStyle('.panel-1-content', 'width', 143, 1);
+  });
+
+  test<MyTestContext>('it respects horizontal minLength (default)', async function (assert) {
+    this.renderController.containerStyle =
+      'max-width: 100%; height: 200px; width: 108px;';
+    await renderHorizontalResizablePanelGroup(this.renderController);
+    assert.hasNumericStyle('.panel-0-content', 'width', 40, 1);
+    assert.hasNumericStyle('.panel-1-content', 'width', 50, 1);
+  });
+
+  test<MyTestContext>('it respects horizontal minLength (length specified)', async function (assert) {
+    this.renderController.containerStyle =
+      'max-width: 100%; height: 200px; width: 108px; border: 1px solid green';
+    this.renderController.panels[0].lengthPx = 45;
+    this.renderController.panels[1].lengthPx = 45;
+    await renderHorizontalResizablePanelGroup(this.renderController);
+    assert.hasNumericStyle('.panel-0-content', 'width', 40, 2);
+    assert.hasNumericStyle('.panel-1-content', 'width', 50, 2);
```

Comment:
can we refer to this as the minLenghtPx of panel2 or something

Reply:
done in 3c5cdac, thanks